plugins/worker/lib/jobs.js

```diff
@@ -346,29 +347,29 @@ async function sendWebhook(configs) {
 
 module.exports = {
     start: {
-        plugins: [Filter, 'Retry', BlockedBy],
+        plugins: [Filter, Plugins.Retry, BlockedBy],
```

Comment:
we have to pass the plugins object instead of plugin name
https://github.com/actionhero/node-resque/releases/tag/v8.2.0